catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/metatype/downloadManager.xml

```diff
@@ -19,7 +19,7 @@
          name="Resource Download Settings"
          id="ddf.catalog.resource.download.ReliableResourceDownloadManager">
         <AD name="Product Cache Directory" id="productCacheDirectory" required="false"
-            type="String" default=""
+            type="String" default="${ddf.home}/data/Product_Cache"
```

Comment:
❗️ Don't think this will work since this namespace the place holder https://github.com/codice/ddf/blob/master/catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml#L25 .